frontend/src/store/modules/oauth.ts

```diff
@@ -16,14 +16,23 @@ const actions = {
     {
       vcsId,
       code,
+      oauthConfig,
     }: {
       vcsId: VCSId;
```

Comment:
oauthConfig?: oauthConfig
vcsId? : VCSId

And put required code field first to be more readable.